trunk/source/ACSLParser/src/de/uni_freiburg/informatik/ultimate/model/acsl/ACSLPrettyPrinter.java

```diff
@@ -212,4 +245,121 @@ public static String binaryOperatorToString(final BinaryExpression.Operator oper
 		case COMPPO, BITVECCONCAT -> throw new AssertionError("Unhandled operator " + operator);
 		};
 	}
+
+	// https://en.cppreference.com/w/c/language/operator_precedence.html
```

Comment:
We should refer to the ACSL standard instead, even though I don't expect much of a difference. In section 2.2.1 there is the following table:
<img width="670" height="503" alt="grafik" src="https://github.com/user-attachments/assets/e348ed71-7aea-42f8-ba61-534edb323f16" />